01-node-tutorial/answers/9-path-module.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path')
+
+console.log(path.sep);
+
+const filePath = path.join('/content', 'subfolder', 'test.tx')
+console.log(filePath);
+
+const base = path.basename(filePath)
+console.log(base);
+
+const absolute = path.resolve(__dirname,'/content', 'subfolder', 'test.tx')
```

Comment:
The second argument here shouldn't have the leading `/`. Adding the leading `/` results in the first part of the path being truncated so the result becomes `/content/subfolder/test.txt` instead of `__dirname/content/subfolder/test.txt`.